python/lsst/sims/maf/batches/visitdepthBatch.py

```diff
@@ -46,56 +48,66 @@ def nvisitsM5Maps(colmap=None, runName='opsim',
         subgroup = 'All visits'
 
     # Set up basic all and per filter sql constraints.
-    sqlconstraints = ['']
-    metadata = ['all bands']
-    if filterlist is not None:
-        sqlconstraints += ['%s = "%s"' % (colmap['filter'], f) for f in filterlist]
-    metadata += ['%s band' % f for f in filterlist]
-
-    # Add additional sql constraint (such as wfdWhere) and metadata, if provided.
-    if (extraSql is not None) and (len(extraSql) > 0):
-        tmp = []
-        for s in sqlconstraints:
-            if len(s) == 0:
-                tmp.append(extraSql)
-            else:
-                tmp.append('%s and (%s)' % (s, extraSql))
-        sqlconstraints = tmp
-        if extraMetadata is None:
-            metadata = ['%s %s' % (extraSql, m) for m in metadata]
-    if extraMetadata is not None:
-        metadata = ['%s %s' % (extraMetadata, m) for m in metadata]
-    metadataCaption = extraMetadata
-    if metadataCaption is None:
-        metadataCaption = 'all visits'
+    filterlist, colors, orders, sqls, metadata = filterList(all=True,
+                                                            extraSql=extraSql,
+                                                            extraMetadata=extraMetadata)
+    # Set up some values to make nicer looking plots.
+    benchmarkVals = mafUtils.scaleBenchmarks(runLength, benchmark='design')
+    # Check that nvisits is not set to zero (for very short run length).
+    for f in benchmarkVals['nvisits']:
+        if benchmarkVals['nvisits'][f] == 0:
+            print('Updating benchmark nvisits value in %s to be nonzero' % (f))
+            benchmarkVals['nvisits'][f] = 1
+    benchmarkVals['coaddedDepth'] = mafUtils.calcCoaddedDepth(
```

Comment:
I think something funny happened here with the line break.
It is currently:
~~~
benchmarkVals['coaddedDepth'] = mafUtils.calcCoaddedDepth(
        benchmarkVals['nvisits'], benchmarkVals['singleVisitDepth’])
~~~

I think `benchmarkVals['nvisits']` should be moved up to line 61 and `benchmarkVals['singleVisitDepth’]` should be on line 62 with the correct alignment.

Reply:
Yes, weird. I suspect this is an artifact from running an autopep8 thing on the original script (which I then copy/pasted here). Fixed.